### DISCUSSION TITLE
Setting location by entering lat long manually

### DISCUSSION THREAD

**Discussion by RoscoeD** [Stellarium/stellarium#2376] [General]
I cannot seem to make this work.  I type in the coordinates and the elevation, type a name into the location box ("My House") and click add to list.  The coordinates change from what I entered.  I've tried this multiple times, slightly varying technique or order, and the lat long numbers still change. Not much, but it shouldn't happen at all.  I wish I had the means to record the screen but I don't.

**Comment by gzotti** [comment]
What version?
If numbers change by "not much", these may be rounding errors. If you don't think so, please give examples for "not much".

**Comment by RoscoeD** [comment]
I originally set Cheyenne WY as my location (41 08 23). I then changed it to my house on the west side of town (41 11 6). I added a name (“My House”) and selected add to list. It changed the coordinates for the new entry back to the original Cheyenne coordinates. 

After an hour of messing with it I finally entered the new coordinates without trying to save them, checked the “use current location as default” and closed the dialog box.   Bottom line it isn’t the amount of change, it’s the fact that what I enter won’t stick if I save it.

**Reply by gzotti** [reply]
OK, not a rounding error then. But which version of Stellarium are you using?

**Comment by RoscoeD** [comment]
Sorry about that.  I’m running  version 0.21.3 (both Windows and macOS).  Both exhibit the same problem.

Note that when I save a custom location, Lat and Long change to what I think is the previously selected location, but elevation changes stick.

**Reply by gzotti** [reply]
We had some changes recently. Can you please try with 0.22.0?

**Comment by RoscoeD** [comment]
Saves properly now, thanks (at least on the MACOS version)

**Comment by RoscoeD** [comment]
Gents, I was able to add a custom horizon to my Windows laptop version of Stellarium following a Youtube Tutorial.  However my desktop is a Mac, and I can find no instructions for adding it (and the folder structure looks nothing like Windows…).  

I did see under “Sky and Viewing Options “ | “Landscapes" Tab an option to add a landscape from a zip archive, so I zipped my custom file (unzipped it would not allow me to select it…odd design choice by you guys) and pointed Stellarium to the zip file, but it gave me an error “No landscape was installed. The selected file is not a ZIP archive or does not contain a Stellarium landscape.”  It’s exactly the same file that worked on the laptop windows version so I’m out of ideas.

Thanks

**Reply by alex-w** [reply]
The Stellarium expected zipped directory with landscape here - please read the Stellarium User Guide

**Reply by gzotti** [reply]
The ZIP interface allows easy distribution of ready-made third-party landscape packages. The User Guide describes what to do if you don't just import a finished working zip file but create your own landscape. If you then have a working landscape, you can zip it for further easy distribution.

**Comment by RoscoeD** [comment]
I’m currently on vacation and away from my iMac, but are you telling me that the MacOS version of stellarium takes a landscape file in a different format then the Windows? Because as I said the same file worked fine in the Windows version.

**Reply by alex-w** [reply]
Same format!

**Reply by gzotti** [reply]
No. You noticed difficulties around ZIP files, which is why we refer to the User Manual for details. You can copy files one-by-one with your operating system, but in-program installation of custom landscapes requires ZIP files, which allows easy distribution of working add-ons.

**Comment by RoscoeD** [comment]
Actually the original email started with how do I get my Windows landscape into a Macos version.  I simply referenced the Landscape import function as something I tried (and failed).  Honestly, I really don’t care about importing a zip file unless that is the only means possible.  I just want my Horizon that works on Windows ini my MacOS version.  Please.

My Windows Stellarium had several options in the landscape folder.  I followed instructions from Youtube to create a new one by copying and editing the config.ini file from one of the “prebuilt” landscapes.  It worked.  I’d like to take that same landscape (apparently a polygonal by the description in the user manual) and port it over to my Mac.  Sounds like from previous emails that the format is identical so this shouldn’t be that hard.  Can I just copy the custom landscape folder from the windows computer and place it in the landscapes folder on the Mac?
Note: Unlike the Windows install, my Mac did not have any data at all in the "$library/application support/Stellarium/landscapes" folder yet there were several I could choose from within the Stellarium app.  That confuses the heck out of me.  Please explain what I’m seeing.
Note2: I read chapter 7 on landscapes in the user guide.  Sorry, but the complexity of that chapter did not impart any useful knowledge upon me (and as an Engineer with graduate degrees I can say with confidence I’m not a dummy)

Thank you for a wonderful product.

Ross

**Reply by gzotti** [reply]
In chapter 5 of the User Guide we explain the difference between installation directory (which users should not modify) and user data directory. On a freshly installed product, the user data directory does not yet contain user-installed data. The structures of the user data directories on Mac, Windows and Linux are identical, and simply copying files from one system to the other should work. 
We are not responsible for third-party instruction videos on YouTube and cannot comment on them. They usually have their own discussion/feedback forum.
